evap/staff/importers.py

```diff
@@ -592,11 +593,13 @@ def create_success_messages(self, students_created, responsibles_created):
         )
 
         if students_created or responsibles_created:
-            message += ":"
-            message += create_user_list_html_string_for_message(students_created)
-            message += create_user_list_html_string_for_message(responsibles_created)
+            message = format_html(
+                "{}: {} {}",
```

Comment:
I don't think `create_user_list_html_string_for_message` ends or begins with a line break, does it?

If so, this would need a line break in between the last two pairs of braces.